docs/style-guide.md

```diff
@@ -44,8 +44,8 @@ const fs = Promise.promisifyAll(require('fs-extra'))
 
 ### Leave a blank line (EOL) at the end of file
 
-### Comments
-Feel free to leave comments where necessary.
+### Code Comments
+Feel free to comment your code when necessary.
```

Comment:
That's right, so we rephrase this to:

`Please feel free to write comments in your code where necessary.`

?